cmd/tsconnect/tsconnect.go

```diff
@@ -22,6 +22,7 @@ var (
 	distDir         = flag.String("distdir", "./dist", "path of directory to place build output in")
 	yarnPath        = flag.String("yarnpath", "../../tool/yarn", "path yarn executable used to install JavaScript dependencies")
 	fastCompression = flag.Bool("fast-compression", false, "Use faster compression when building, to speed up build time. Meant to iterative/debugging use only.")
+	controlURL      = flag.String("control-url", "", "URL of the Tailscale control server. If empty, defaults to DefaultControlURL.")
```

Comment:
Document that this only works in dev mode

Reply:
Someday this might be used by a customer on the US Compliance server, or one of the onprem sites.

Reply:
Sure, but as implemented it only works in dev mode

Reply:
Better yet fail with an error if it's used in other modes.

Reply:
But also document, or maybe rename to `--dev-control`?